doc/en/changelog.rst

```diff
@@ -504,6 +504,8 @@ Features
   such as default values or which set of command line options to add.
 
 
+- `#6906 <https://github.com/pytest-dev/pytest/issues/6906>`_: ``--code-highlight`` decides whethere code should be highlighted in terminal output.
+
```

Comment:
we manage changelogs for new features by adding a changelog fragment.  You can see how these are set up by looking at the `./changelog` directory (there's a bunch of files in there already that you can base yours on).  The [CONTRIBUTING](https://github.com/pytest-dev/pytest/blob/master/CONTRIBUTING.rst) guide also covers changelogs as well